test/kernel_acl.js

```diff
@@ -194,6 +195,36 @@ contract('Kernel ACL', accounts => {
             })
         })
 
+        context('removing managership', () => {
+            const newManager = accounts[4]
+            assert.notEqual(newManager, granted, 'newManager should not be the same as granted')
+
+            beforeEach(async () => {
+                const receipt = await acl.removePermissionManager(app, role, { from: granted })
+                assertEvent(receipt, 'ChangePermissionManager')
+            })
+
+            it('removes manager', async () => {
+                const noManager = await acl.getPermissionManager(app, role)
+                assert.equal('0x0000000000000000000000000000000000000000', noManager, 'manager should have been removed')
+            })
+
+            it('can recreate permission', async () => {
+                const createReceipt = await acl.createPermission(newManager, app, role, newManager, { from: permissionsRoot })
+                assertEvent(createReceipt, 'SetPermission')
+                assertEvent(createReceipt, 'ChangePermissionManager')
+
+                const grantReceipt = await acl.grantPermission(granted, app, role, { from: newManager })
+                assertEvent(grantReceipt, 'SetPermission')
+            })
+
+            it('old manager lost power', async () => {
+                return assertRevert(async () => {
+                    await acl.grantPermission(newManager, app, role, { from: granted })
```

Comment:
Wouldn't it be better to use another account rather than `newManager` here? Otherwise there are actually 2 reasons to fail here: `granted` not being the manager and `newManager` already having the permission, right?

Reply:
Good point!